src/app/shared/constants/constants.ts

```diff
@@ -148,3 +149,12 @@ export class ModeConstants {
 }
 
 export const EMPTY_RESULT = { totalAmount: 0, entities: [] };
+
+export const CustomMatTooltipDefaultOptions: MatTooltipDefaultOptions = {
```

Comment:
Give it a more specific name. "Custom" is a very general word. Maybe something like "NoInteractTooltipOptions".